roles/varnish/templates/etc/varnish/varnish.vcl

```diff
@@ -318,14 +312,14 @@ sub vcl_recv {
     }
     
     if (req.method == "PURGE") {
-        if (!client.ip ~ purge) {
+        if (!std.ip(regsub(req.http.x-forwarded-for, ",.*$", ""), "0.0.0.0") ~ purge) {
```

Comment:
took me a second to understand this, but I _think_ it makes sense now …

Reply:
I'll add some comments